### DISCUSSION TITLE
Merge from upstream

### DISCUSSION THREAD

**Discussion by Bertaz** [itzAlex/chatterino7#15] [General]
Is there any conflict that's blocking the merge from either chatterino2 or chatterino7?
I see Pr #14 is open from October and last commit to this repo is from August. Considering chatterino [2.4.0 release](https://github.com/Chatterino/chatterino2/releases/tag/v2.4.0) it would be nice to merge

**Comment by itzAlex** [comment]
> Is there any conflict that's blocking the merge from either chatterino2 or chatterino7? I see Pr #14 is open from October and last commit to this repo is from August. Considering chatterino [2.4.0 release](https://github.com/Chatterino/chatterino2/releases/tag/v2.4.0) it would be nice to merge

I plan to update this fork starting in February (I am in exams right now!). There are a lot of conflicts with that pull request 😞 I have to check that everything new works with what is currently implemented on our side.

**Comment by itzAlex** [comment]
@Bertaz The new update is now available, sorry for the delay!
